profiles/common/modules/custom/multisite_create_button/multisite_create_button.module

```diff
@@ -27,6 +31,11 @@ function multisite_create_button_block_view($delta = '') {
       $block['subject'] = t('Create content');
       $block['content'] = _multisite_create_button_contextual_links();
       break;
+
+    case 'og-contextual-links':
+      $block['subject'] = t('Create content');
+      $block['content'] = _multisite_og_button_contextual_links();
```

Comment:
this should be renamed into _multisite_create_button_contextual_links()

Reply:
_multisite_create_button_contextual_links is the one used for the previous "create_button" and this function is different, so I renamed it as _multisite_create_button_og_contextual_links()